frappe/utils/background_jobs.py

```diff
@@ -519,3 +522,24 @@ def get_job(job_id: str) -> Job:
 		return Job.fetch(create_job_id(job_id), connection=get_redis_conn())
 	except NoSuchJobError:
 		return None
+
+
+BACKGROUND_PROCESS_NICENESS = 10
```

Comment:
This can be debated, idk what's "IDEAL" number here, but assuming default of 0, and background processes of `10`:

- Web processes will get X time with `0` niceness
- Background processes will get 0.5 * X with `10` nicness. (20-10) / (20-0)